johnsonandjohnson/Bodiless-JS
# feat(migration-tool): Enhance handling of non-existing source site resources

## Changes


- Skip page download resource with name 404 to avoid build conflict during migration process.
- Display warning message on non-existing source download, including 400 and 500 errors.


## Test Instructions

- Configure the migration-settings.json for site to be migrated. The site url should include a page with non-existing source, i.e. missing image file, or PDF file.
- Run `npm run migrate`

Expected:

- No 404 file created for non-existing source.
- Console output shows non-existing urls.


## Related Issues


Fixes #202
